milestone-3/frontend/src/App.js

```diff
@@ -0,0 +1,31 @@
+import styles from "./App.css";
+import About from "./components/About";
+import RecipePage from "./components/RecipePage";
+import Home from "./components/Home";
+import { Routes, Route, BrowserRouter, Link, Outlet } from "react-router-dom";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <header>
+        <h1>bora's bakery 🍞</h1>
+        <nav>
+          <Link className={styles.link} to='/'>
+            home
+          </Link>
+          <Link className={styles.link} to='about'>
+            about me
+          </Link>
+        </nav>
+        <Outlet />
+      </header>
```

Comment:
So this stuff in lines 11 through 21 could be better abstracted into a Navbar component. This is technically one of the milestone 3 requirements, but looking at your other components I can see you understand the concepts well. If you just want to turn this into a navbar component and turn it in with milestone 4 that should be good, its a pretty easy component similar to your home and about me ones.